doc/user/advanced_constraints.rst

```diff
@@ -344,6 +347,14 @@ For specifics of the mathematical formulation of the available group constraints
 
 .. seealso:: The :ref:`built-in national-scale example <examplemodels_nationalscale_settings>`'s ``scenarios.yaml`` shows two example uses of group constraints: limiting shared capacity with ``energy_cap_max`` and enforcing a minimum shared power generation with ``supply_share_min``.
 
+
+``demand_share_per_timestep_decision``
+--------------------------------------
+
+The ``demand_share_per_timestep_decision`` constraint is a special case amongst group constraints, as it introduces a new decision variable, allowing the model to set the share of demand met by each technology given in the constraint's group, across the locations given in the group. The fraction set in the constraint is the fraction of total demand over which the model has control. Setting this to anything else than ``1.0`` only makes sense when a subset of technologies is targeted by the constraint.
```

Comment:
Mention the use of `.inf` in cases where there are other sources of carrier consumption, other than demand: using conversion techs, losses from storage/transmission